Point_set_processing_3/include/CGAL/estimate_lfs.h

```diff
@@ -582,10 +582,10 @@ template <typename ConcurrencyTag,
           typename NamedParameters = parameters::Default_named_parameters>
 void
 estimate_local_feature_size(PointRange& points,
+                            LfsMap lfs_map,
```

Comment:
We have put it at the end, so that first come input parameters, then the output parameter, and finally the optinal named parameters.